wsd/ClientSession.cpp

```diff
@@ -85,7 +85,7 @@ ClientSession::ClientSession(
                                    << "], current number of connections: " << curConnections);
 
     // populate with random values.
-    for (auto it : _clipboardKeys)
+    for (size_t i = 0; i < N_ELEMENTS(_clipboardKeys); ++i)
```

Comment:
I assume that here `const auto& it` would not be enough, would introduce some unused variable warning or so.

Reply:
yeah, exactly, and everything else range-based is just a miserable hackaround